core/src/main/java/com/linecorp/armeria/server/Route.java

```diff
@@ -95,6 +95,11 @@ default RoutingResult apply(RoutingContext routingCtx) {
      */
     RoutingResult apply(RoutingContext routingCtx, boolean isRouteDecorator);
 
+    /**
+     * Returns the {@link PathMapping} of this {@link Route}.
+     */
+    PathMapping pathMapping();
```

Comment:
Removed this method and fixed the following lines:
https://github.com/line/armeria/pull/3340/files#diff-d94019f9ef8458d0b4a04a1be6e2a6ebaf9f85cbab92e2ac6bc6bc42f89f0d2bR81-R96

Is it okay if I use `CatchAllPathMapping.INSTANCE` directly from there? I'm not sure, but I think the fallback route's `PathMapping` must be `CatchAllPathMapping` because `/a/`'s fallback path is always `/a`. So I think it would be better to use `CatchAllPathMapping.INSTANCE` rather than making `PathMapping` public. Please let me know what you think. 🙇‍♂️